README.md

```diff
@@ -1,3 +1,91 @@
 # opentelem-python
 
-Open Telemetry Codecov Python Prototype
+_Note: This package is part of the [Runtime Insights Early Access Program](https://about.codecov.io/product/feature/runtime-insights/)._
+
+## Purpose
+
+This package allows Python projects to leverage Codecov's [Runtime Insights](https://about.codecov.io/product/feature/runtime-insights/) feature.
+
+## Requirements and Pre-requisites
+
+1. A repository that is active on [Codecov](https://codecov.io)
+2. A profiling token for that repository, obtainable from Codecov.
+3. Python version >=3.6
+
+## Installation
+
+First, install the package:
+
+```
+pip install codecovopentelem
+```
+
+Second, include the following snippet in your application's startup / bootstrapping process:
+
+```python
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    BatchSpanProcessor,
+    SimpleSpanProcessor,
+)
+from opentelemetry import trace
+
+from codecovopentelem import (
+    CoverageSpanFilter,
+    get_codecov_opentelemetry_instances,
+)
+from utils.time import format_time
+
+provider = TracerProvider()
+trace.set_tracer_provider(provider)
+
+"""
+CONFIGURATION
+"""
+current_version = "your-application-version"
+current_env = "your-application-envrionment"
+export_rate = 10
+untracked_export_rate = 0
+profiling_token="your-profiling-token"
+
+
+generator, exporter = get_codecov_opentelemetry_instances(
+    repository_token=profiling_token,
+    version_identifier=current_version,
+    sample_rate=export_rate,
+    filters={
+        CoverageSpanFilter.regex_name_filter: None,
```

Comment:
There is no need for this. This is already the default. Maybe we can document what  they can do if they want to customize this instead?

Reply:
Can you provide all the known defaults? If so I can update the configuration section of the README with the defaults and remove / edit the code portion of the doc accordingly.